examples/employee-app/src/main/public/EmployeeController.js

```diff
@@ -0,0 +1,129 @@
+/* Copyright 2018 Oracle and/or its affiliates. All rights reserved. */

```

Comment:
This should use the proper copyright header.

Reply:
Waiting on our client fix to see if we keep this file.

Reply:
Done.